web/package.json

```diff
@@ -116,5 +117,11 @@
       "src/serviceWorker.ts",
       "src/app/store.ts"
     ]
+  },
+  "overrides": {
+    "typescript": "^5.0.0"
+  },
+  "resolutions": {
+    "pmtiles": "2.7.2"
```

Comment:
Do we need this because `ol-pmtiles` tries to pull in an older version of `pmtiles`?

Reply:
I was running into really odd dependency resolution issues and this is another artifact. The WetPaddlers project had pmtiles as a dependency, so I originally had it as a dependency of our web project as well. The ol-pmtiles dependency had pmtiles as a child dependency. I was running into issues with both pmtiles 2.7.2 and 2.10.0 being installed and the resolution allowed me to workaround it. Now that I've removed pmtiles as a dependency of our web project, things seem to work without the resolution!